lms/templates/ccx/schedule.html

```diff
@@ -32,7 +32,7 @@
 <section id="enter-date-modal" class="modal" aria-hidden="true">
   <div class="inner-wrapper" role="dialog">
     <button class="close-modal">
-      <i class="fa-remove"></i>
+      <i class="fa fa-times"></i>
```

Comment:
Is that ``fa`` class supposed to be there? What does it do?

Although using an icon named ``fa-times`` confuses me in this case, I've confirmed that it's common practice at edX.